packages/vats/test/test-boot-config.js

```diff
@@ -0,0 +1,61 @@
+// @ts-check
+import { test } from '@agoric/swingset-vat/tools/prepare-test-env-ava.js';
+
+import { promises as fsPromises } from 'fs';
+import path from 'path';
+
+import { fit, M } from '@agoric/store';
+import { ParametersShape as BootParametersShape } from '../src/core/boot-psm.js';
+
+const pathname = new URL(import.meta.url).pathname;
+const dirname = path.dirname(pathname);
+const asset = (...ps) =>
+  fsPromises.readFile(path.join(dirname, ...ps), 'utf-8');
+
+/**
+ * NOTE: this pattern suffices for PSM bootstrap,
+ * but does not cover the whole SwingSet config syntax.
+ *
+ * {@link packages/SwingSet/docs/configuration.md}
+ * TODO: move this to swingset?
+ *
+ * @see SwingSetConfig
+ * in packages/SwingSet/src/types-external.js
+ */
+const shape = (() => {
```

Comment:
why the function? oh, to make a scope? these should be in their own `typeGuards.js` file which can encapsulate.

I think this does make more sense in SwingSet. prolly `vats/typeGuards.js`

Reply:
ok. let's do this in #6205 in any case.